tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -147,10 +147,10 @@ COMMON_FLAGS := \
   -ffunction-sections \
   -fdata-sections \
   -fmessage-length=0 \
-  -DTF_LITE_STATIC_MEMORY \
```

Comment:
This removal is a bit surprising, can you add details about what functions are needed.

Reply:
for example GetIntermediatesSafe,TfLiteIntArrayCopy,TfLiteIntArrayCreate,GetScratchSafe thise functions are called from lstm and are defined only if undef of TF_LITE_STATIC_MEMORY flag

Reply:
@petewarden 
added TF_LITE_STATIC_MEMORY  flag as it is in original

Reply:
Thanks! Could you move it so that it's after -fmessage-length=0 \ as it was originally? That will help reduce noise in the diff.